### PR TITLE
Subtask/as 1206 fwa nojs pick

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

AS-1206

## Description of change
forms web app implementation for cookie banner sans js

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: e2e-tests/cypress/integration/cookies-consent-bar.feature [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-    Given a user has not previously submitted cookie preferences
-    When the user accepts not necessary cookies
-    Then the GA cookies are enabled
-
-  @as-98 @as-98-3
-  Scenario: Reject Not necessary Cookies
-    Given a user has not previously submitted cookie preferences
-    When the user rejects not necessary cookies
-    Then the GA cookies remain disabled
+#  @as-98 @as-98-2
```
Comment (a6software) [inline e2e-tests/cypress/integration/cookies-consent-bar.feature:13]: @rcmsoftwaredevelopment please uncomment this one and ensure it passes as part of your current ticket - should need no work, but ping me if it does

File: e2e-tests/cypress/integration/common/cookies.js [hunk 2]
```diff
@@ -57,13 +56,11 @@ When('the user visits the service', () => {
...
 Then('not necessary cookies are disabled', () => {
-  const expectedCookiePolicy = JSON.stringify(DEFAULT_COOKIE_POLICY);
-
   cy.getCookies()
-    .should('have.length', 2)
+    .should('have.length', 1)
     .then((cookies) => {
       expectExpressSessionCookieIsDefined(cookies);
-      expectCookiePolicy(cookies, expectedCookiePolicy);
+      expectCookieIsNotDefined(cookies, 'cookie_policy');
```
Comment (a6software) [inline e2e-tests/cypress/integration/common/cookies.js:63]: feels weird to remove this but it's in line with the AC. This is going right back in with the JS banner variant of this feature.

File: packages/forms-web-app/src/app.js [hunk 3]
```diff
@@ -106,7 +105,6 @@ app.use(
   express.static(path.join(__dirname, '..', 'node_modules', 'govuk-frontend', 'govuk', 'all.js'))
 );
 app.use(fileUpload(config.fileUpload));
-app.use(initialCookiesMiddleware(env));
```
Comment (a6software) [inline packages/forms-web-app/src/app.js:109]: these will be added back in for the JS variant up next.

**Review by frank-dodion — COMMENTED** [review]
I think this would be good to get into the feature branch.

**Review by frank-dodion — APPROVED** [review]

**Pull request merged**